onnxruntime/core/framework/tensorprotoutils.cc

```diff
@@ -1266,17 +1250,19 @@ common::Status ConstantNodeProtoToTensorProto(const ONNX_NAMESPACE::NodeProto& n
 }
 
 common::Status ConstantNodeProtoToTensorProto(const ONNX_NAMESPACE::NodeProto& node,
-                                              const Path& model_path,
+                                              const std::filesystem::path& model_path,
                                               ONNX_NAMESPACE::TensorProto& tensor) {
   return ConstantNodeProtoToTensorProto(node, model_path, tensor, node.output(0));
 }
 
 #if !defined(DISABLE_SPARSE_TENSORS)
 static Status CopySparseData(size_t n_sparse_elements,
                              const ONNX_NAMESPACE::TensorProto& indices,
-                             const Path& model_path,
-                             gsl::span<const int64_t> dims,
-                             std::function<void(size_t from_idx, size_t to_idx)> copier) {
+                             const std::filesystem::path& model_path,
+                             gsl::span<const int64_t>
+                                 dims,
```

Comment:
why are there so many formatting changes? the PR is already kind of big.

Reply:
Previously a lot of the lines are longer than 120 characters, I wrapped them. 
Some of the lines were wrapped arbitrarily before reaching the limit, I updated the line wrapping position. 
Also, I put all the short "return" statement in a separated line for easing debugging. It would be easier to setup breaking point at these locations. (If I remember correctly, it was a suggestion from Scott from years ago. But it was just a recommendation, not a strict requirement so not all our code follows that).

Reply:
For example, the line 1310 had 146 chars
```
ORT_RETURN_IF_NOT(indices.int32_data_size() == static_cast<int64_t>(elements), "Sparse indices int32 data size does not match expected");
```